commands/certmanager/constants.go

```diff
@@ -0,0 +1,37 @@
+package certmanager
+
+// flags
```

Comment:
Flags should have `Flag` pre-pended to their variable name (or `flag` if unexported)

Like `FlagCertId`